cli/test/lib/tasks/state_spec.js

```diff
@@ -95,99 +134,176 @@ describe('lib/tasks/state', function () {
     })
 
     it('resolves path to binary/installation from version', function () {
-      expect(state.getBinaryDir('4.5.6')).to.be.equal(path.join(cacheDir, '4.5.6', 'Cypress.app'))
+      expect(state.getBinaryDir('4.5.6')).to.be.equal(
+        path.join(cacheDir, '4.5.6', 'Cypress.app')
+      )
     })
 
     it('rejects on anything else', function () {
       os.platform.returns('unknown')
-      expect(() => state.getBinaryDir().to.throw('Platform: "unknown" is not supported.')
+      expect(() => {
+        return state.getBinaryDir().to.throw('Platform: "unknown" is not supported.')
+      }
       )
     })
   })
 
   context('.getBinaryVerifiedAsync', function () {
     it('resolves true if verified', function () {
       sinon.stub(fs, 'readJsonAsync').resolves({ verified: true })
-      return state.getBinaryVerifiedAsync('/asdf')
-      .then((isVerified) => expect(isVerified).to.be.equal(true))
+
+      return state
+      .getBinaryVerifiedAsync('/asdf')
+      .then((isVerified) => {
+        return expect(isVerified).to.be.equal(true)
+      })
     })
     it('resolves undefined if not verified', function () {
-      sinon.stub(fs, 'readJsonAsync').rejects({ code: 'ENOENT' })
-      return state.getBinaryVerifiedAsync('/asdf')
-      .then((isVerified) => expect(isVerified).to.be.equal(undefined))
+      const err = new Error()
```

Comment:
the actual test that has changed (the rest is prettier and eslint running on these JS files)